oar/core/notification_mgr.py

```diff
@@ -443,6 +459,45 @@ def get_slack_message_for_dropped_and_must_verify_bugs(
 
         return message
 
+    def get_slack_message_for_docs_and_prodsec_approve(
+        self, doc_appr, prodsec_appr
+    ):
+        """
+        manipulate slacke message for dropped bugs and must verified bugs
+
+        Args:
+            doc_appr ([]): list of no doc approved advisories
+            prodsec_appr ([]): list of no product security approved advisories
+
+        Returns:
+            str: slack message
+        """
+        gid = self.sc.get_group_id_by_name(
+            self.cs.get_slack_user_group_from_contact(
+                "art", "doc_id"
+            )
+        )
+        userid = self.cs.get_slack_user_id_from_email("art","secu_email")
+        message = ""
+        if len(doc_appr)&len(prodsec_appr):
+            logger.info(
+                f"notfiy for Docs and Prodsec approval"
+                )
+            message = f"[{self.cs.release}] Hello {gid}, please approve Docs for advisories:{str(doc_appr)} and {userid} Prodsec for advisories:{str(prodsec_appr)}, thanks!"
```

Comment:
it's better to add z-stream release version at the beginning. let's avoid using `Please` , instead `Could you approve xxx`